lib/components/AlbumScreen/preset_chip.dart

```diff
@@ -54,24 +55,21 @@ class _PresetChipsState extends State<PresetChips> {
     offset = min(max(0, offset),
         widget.chipWidth * (widget.values.length) - maxWidth - _spacing);
 
-    _controller.animateTo(
-      offset,
-      duration: Duration(milliseconds: 350),
-      curve: Curves.easeOutCubic,
-    );
+    if (_controller == null) {
+      _controller = ScrollController(initialScrollOffset: offset);
+    } else {
+      _controller!.animateTo(
```

Comment:
why is this an if/else and not just a `_controller =?? ScrollController(initialScrollOffset: offset);`  
Don't we wanna scroll if there's no scroll controller yet?

Reply:
No, if _controller is null we're doing the first build of this widget so we don't want to animate the scroll, we want the offset already applied before the widget shows.